packages/api-clients/graphql-client/src/graphql-client/types.ts

```diff
@@ -79,6 +80,7 @@ export interface ClientOptions {
   headers: HeadersObject;
   url: string;
   customFetchApi?: CustomFetchApi;
+  keepalive?: boolean;
```

Comment:
I believe we  don't need this, we just need to add to the Request?

Reply:
Good catch! Done.